dapp/src/components/buySell/SwapHomepage.js

```diff
@@ -346,8 +346,8 @@ const SwapHomepage = ({
         }
       )
       setStoredCoinValuesToZero()
-      setSelectedBuyCoinAmount(0)
-      setSelectedRedeemCoinAmount(0)
+      setSelectedBuyCoinAmount('')
```

Comment:
@sparrowDom I was going to ask for this since it currently resets to "0" instead of the empty state. I assume that this commit just hasn't been deployed.

Reply:
Yup that is correct, have done this today, but not deployed it to staging